sci-physics/rivet/rivet-2.6.0.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
Did you run repoman? It would have complained at least about the obsolete header :-)